### PR TITLE
Add Pipeline Hint

### PR DESCRIPTION
**Pull request by SimonCMoore**
#### Which issue(s) does this change fix?
Adds hint to show users how to use the new recently added create pipeline configuration

#### Checklist

- [ ] Add input/output [type hints](https://docs.python.org/3/library/typing.html) to new functions/methods
- [ ] Write design document ([Do I need to write a design document?](https://github.com/aws/aws-sam-cli/blob/develop/DEVELOPMENT_GUIDE.md#design-document))
- [ ] Write unit tests
- [ ] Write/update functional tests
- [ ] Write/update integration tests
- [ ] `make pr` passes
- [ ] `make update-reproducible-reqs` if dependencies were changed
- [ ] Write documentation

By submitting this pull request, I confirm that my contribution is made under the terms of the [Apache 2.0 license](https://www.apache.org/licenses/LICENSE-2.0).

### CODE REVIEW

File: samcli/commands/init/interactive_init_flow.py [hunk 1]
```diff
@@ -143,8 +143,12 @@ def _generate_from_app_template(
     Application Template: {app_template}
     Output Directory: {output_dir}
     
-    Next steps can be found in the README file at {output_dir}/{name}/README.md
-        """
+   Next application steps can be found in the README file at {output_dir}/{name}/README.md
```
Comment (aahung) [inline samcli/commands/init/interactive_init_flow.py:146]: indentation is off here

File: samcli/commands/init/interactive_init_flow.py [hunk 2]
```diff
@@ -156,8 +160,12 @@ def _generate_from_app_template(
     Dependency Manager: {dependency_manager}
     Output Directory: {output_dir}
 
-    Next steps can be found in the README file at {output_dir}/{name}/README.md
-        """
+     Next application steps can be found in the README file at {output_dir}/{name}/README.md
```
Comment (aahung) [inline samcli/commands/init/interactive_init_flow.py:163]: indentation is off

**Review by aahung — APPROVED** [review]
LGTM except the indentations

**Comment by SimonCMoore** [comment]
Indentation fixed
image

**Review by c2tarun — APPROVED** [review]

File: samcli/commands/init/interactive_init_flow.py [hunk 3]
```diff
@@ -156,8 +160,12 @@ def _generate_from_app_template(
...
     Output Directory: {output_dir}
 
-    Next steps can be found in the README file at {output_dir}/{name}/README.md
-        """
+    Next application steps can be found in the README file at {output_dir}/{name}/README.md
+    
+    Commands you can use next
+    =========================
+    [*] Create pipeline: cd {name} && sam pipeline init --bootstrap
+    """
```
Comment (mndeveci) [inline samcli/commands/init/interactive_init_flow.py:168]: Small UX nit, we usually print next steps in yellow do we want to follow same pattern here?

**Comment by SimonCMoore** [comment]
image

Updated

**Review by mndeveci — APPROVED** [review]

File: samcli/commands/init/interactive_init_flow.py (hunk 3 above)
Comment (SimonCMoore) [reply samcli/commands/init/interactive_init_flow.py]: Thanks updated to use yellow, screenshot in comments

**Pull request merged**